packages/devtools_app/lib/src/screens/profiler/cpu_profile_model.dart

```diff
@@ -1261,18 +1269,21 @@ class _CpuProfileTimelineTree {
     return null;
   }
 
-  String? get resolvedUrl => isCodeTree
+  String? get resolvedUrl => isCodeTree && _function is vm_service.FuncRef?
       ?
       // TODO(bkonyi): not sure if this is a resolved URL or not, but it's not
       // critical since this is only displayed when VM developer mode is
       // enabled.
-      _function?.location?.script!.uri
+      (_function as vm_service.FuncRef?)?.location?.script?.uri
       : samples.functions![index].resolvedUrl;
 
   int? get sourceLine {
     final function = _function;
     try {
```

Comment:
what about the case described in the comment: "or generally if `function.location.line` throws an exception."? This would still apply even if we are explicitly checking for FuncRef